Bottom: 9c11e46e9e89a7fe90f96947c1b5f89428dcacf6
Top:    4d91d69621a4849cbb1008e0222430122af2d989
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-24 02:06:45 +0100

Refresh of draw-c-change-drawtop-to-drawo

---

diff --git a/src/draw.c b/src/draw.c
index acf2242..7345912 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -589,7 +589,7 @@ DrawPPV (int group, const BoxType *drawn_area)
 
       if (group == solder_group)
         {
-          side = COMPONENT_LAYER;
+          side = SOLDER_LAYER;
           r_search (PCB->Data->pad_tree, drawn_area, NULL, pad_callback, &side);
         }
     }
